packages/react/src/components/Button/Button.js

```diff
@@ -16,33 +16,34 @@ import { usePrefix } from '../../internal/usePrefix';
 import { useId } from '../../internal/useId';
 import toggleClass from '../../tools/toggleClass';
 import { useFeatureFlag } from '../FeatureFlags';
+import * as FeatureFlags from '@carbon/feature-flags';
 
 const Button = React.forwardRef(function Button(
   {
-    children,
     as,
+    children,
     className,
-    disabled,
-    small,
-    size,
-    kind,
+    dangerDescription = 'danger',
+    disabled = false,
+    hasIconOnly = false,
     href,
-    isExpressive,
-    isSelected,
-    tabIndex,
-    type,
-    renderIcon: ButtonImageElement,
-    dangerDescription,
     iconDescription,
-    hasIconOnly,
-    tooltipPosition,
-    tooltipAlignment,
-    onClick,
+    isExpressive = false,
+    isSelected,
+    kind = 'primary',
     onBlur,
+    onClick,
     onFocus,
     onMouseEnter,
     onMouseLeave,
-    ...other
+    renderIcon: ButtonImageElement,
+    size = 'lg',
```

Comment:
Should this be `md`? This might be why VRT is failing in Percy


https://user-images.githubusercontent.com/3360588/151428614-95424108-0fc1-4d26-af0f-319c0e30f771.mov

Reply:
I went with the design guidance on the [v11 site](https://v11.carbondesignsystem.com/components/button/usage/#button-sizes). Did i misinterpret that?

Reply:
Ah, I think in v10 the default needs to be `md`, for v11 the default can be `lg`. I believe the value for v10 `md` is the same as v11's `lg`. As the PR stands now, the storybook for v10 looks wrong, but the v11 storybook looks correct. Percy only runs against v10 code right now.

Reply:
That did it! Thx!